.github/workflows/release_tests.yml

```diff
@@ -25,6 +26,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install pyhf[tensorflow,torch,minuit,xmlio]
+        python -m pip install 'pytest~=3.5' pytest-cov
```

Comment:
why is `pytest-cov` not installed as part of the development extra?

Reply:
As in why not do 

```
python -m pip install pyhf[tensorflow,torch,minuit,xmlio,develop]
```
?

We could do that. The main reason for not just doing `python -m pip install pyhf[complete]` is to be very explicit what is actually being tested in release tests.

Reply:
I prefer this way, because we need to check if everything (except `develop`) is necessary for performing the tests. We have a lot of extra stuff that doesn't need to be checked for user-facing API usage.

Reply:
that's a good reason I guess

Reply:
If https://github.com/scikit-hep/pyhf/pull/648 had gone in before release `v0.2.2` then I would say this line would just be

```
python -m pip install pyhf[test]
```

(maybe something to keep in mind for the next release). However, as it didn't, then I'm trying to be more explicit for the time being.